pom.xml

```diff
@@ -299,7 +299,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-source-plugin</artifactId>
-        <version>2.1.2</version>
+        <version>3.0.1</version>
```

Comment:
Latest is `3.1.0`, though I see that was just released a couple of weeks ago.

https://search.maven.org/search?q=g:org.apache.maven.plugins%20AND%20a:maven-source-plugin&core=gav

Reply:
Should be `3.1.0` then.